src/components/marker-table/index.js

```diff
@@ -129,6 +128,10 @@ class MarkerTree {
           case 'Network':
             category = 'Network';
             break;
+          case 'Text':
+            category = name;
+            name = data.name;
+            break;
```

Comment:
I looked closer to how this marker behaves. Then here is what I suggest:
* use `category = 'Text'`
* for `name` concatenate the name and `data.name`, maybe like this:
   ```js
name += ` [${data.name}]`
```